### ISSUE TITLE
__workerManifest.assets is missing entry for manifest-{version}.js

### ISSUE THREAD

**Issue by aschultz** [remix-pwa/monorepo#240]
Remix generates a file like `assets/manifest-12345.js` that contains the list of routes and their dependencies, and links to this from the HTML doc. But RemixPWA isn't including it in the __workerManifest.assets list, which makes it tricky to ensure that file gets cached for offline use.

I think maybe the PWA vite plugin that generates the assets list is running before Remix generates the manifest file?

**Comment by ShafSpecs** [comment]
Thanks for bringing this to attention! I would take a look when I am free 👍

**Comment by aschultz** [comment]
Any updates on this?

**Comment by ShafSpecs** [comment]
Yep, thx for bringing it to my attention, this is slated for release alongside v5. I would have to manually include this, but this is a weird issue that it doesn't show up: https://github.com/remix-pwa/monorepo/blob/6fa72a544991a59f8b384a733746a6ec65af75f5/packages/dev/src/plugins/virtual-sw.ts#L231-L246

Do note that in dev, Remix PWA uses the public folder as the build folder, so it doesn't bundle anything in `build` - building it in non-dev mode switches the focus to `build/client`

**Comment by Abhimanyu08** [comment]
has this been resolved?
